pls/rootchain_manager.go

```diff
@@ -164,8 +166,9 @@ func (rcm *RootChainManager) Start() error {
 }
 
 func (rcm *RootChainManager) Stop() error {
-	rcm.txManager.Stop()
 	rcm.backend.Close()
+	rcm.txManager.Stop()
+	rcm.wg.Wait()
```

Comment:
`rcm` and `rcm.txManager` doesn't need to share `WaitGroup` at all. They can use their own `WaitGroup`. Besides, current `rcm` doesn't need `WaitGroup` so we can make `WaitGroup` only when the `tx.Manager` is created

Reply:
Updated at 656e43d